apiml-package/build.gradle

```diff
@@ -0,0 +1,81 @@
+ext {
+    artifactName = 'apiml-package'
+    pullNo = project.hasProperty("pullRequest") && project.getProperty("pullRequest") != null ? "-" + project.getProperty("pullRequest") : ""
+}
+
+configurations {
+    zaasServiceJar
+    zip
+}
+def resourceDir = 'src/main/resources/'
+
+dependencies {
+    zaasServiceJar(project(path: ":apiml", configuration: "liteJarConfiguration"))
+}
+
+task packageApiml(type: Zip) {
+    archiveFileName = artifactName + pullNo + ".zip"
+    includeEmptyDirs = true
+
+    into('/') {
+        from "$buildDir/convert/manifest.yaml"
+        from "../schemas/zaas-schema.json"
+    }
+
+    into('bin/') {
+        from configurations.zaasServiceJar
+        from "$resourceDir/bin/start.sh"
+        from "$resourceDir/bin/validate.sh"
+    }
+}
+
+// TODO should z/OSMF static definition be part of this package now?
```

Comment:
todo, should it be addressed now?

Reply:
yes, I'll pick this one up now that I switch to install-packaging